### PR TITLE
Optic

### PR DESCRIPTION
**Pull request by roveri-marco**
This PR will close issue #45  (adding support for optic) and also issue #46 (putting -e to all the singularity run to avoid use of host environment variables)

### CODE REVIEW

File: planutils/packages/optic/manifest.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "The OPTIC temporal planner",
+    "description": "The OPTIC temporal planner (see https://github.com/roveri-marco/optic)",
+    "install-size": "297MB",
```
Comment (haz) [inline planutils/packages/optic/manifest.json:4]: Bit of a heavy hitter! Already put in the tips/tricks from the wiki on making it small? We've other planners that clock in at this size, so I'm assuming so, but just want to make sure.

File: planutils/packages/optic/uninstall [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+rm roveri-marco-optic.latest.sif
```
Comment (haz) [inline planutils/packages/optic/uninstall:3]: Shouldn't this be just `optic.sif` given the install script above?

**Review by haz — CHANGES_REQUESTED** [review]
Thanks @roveri-marco ! Just a few comments left...

**Comment by roveri-marco** [comment]
I will revise later. Thanks for the feedback.

File: planutils/packages/optic/install [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+# The install script will be run from the package's directory
+
+# To use if root is required
+#[ "$UID" -eq 0 ] || (echo "installation requires root access"; exec sudo "$0" "$@")
+
+# Recipe for singularity images
+## Fetch the image
+#singularity pull --name <image name> <singularity shub url>
+
+singularity pull --name optic.sif https://github.com/roveri-marco/optic/releases/download/1.0/roveri-marco-optic.latest.sif
```
[the remark this answers is not in the document]
Comment (haz) [reply planutils/packages/optic/install:14]: Will pull it in as is for now -- there may be no easy way in the end :P.

**Review by haz — APPROVED** [review]
Thanks!

**Pull request merged**